DataConnectors/Senserva/SenservaPro.json

```diff
@@ -0,0 +1,159 @@
+{
+	"id": "SenservaPro",
+	"title": "SenservaPro",
+	"publisher": "Senserva",
+	"descriptionMarkdown": "The SenservaPro data connector provides a viewing experience for your SenservaPro scanning logs. View dashboards of your data, use queries to hunt & explore, and create custom alerts for your needs. Start your security journey today",
+	"additionalRequirementBanner": "No additional requirements needed",
+	"graphQueries": [
+		{
+			"metricName": "Total data received",
+			"legend": "SenservaPro",
```

Comment:
Legend name should be data type name. Please update the legend to SenservaPro_CL.

Thanks,
Aditi Maheshwari

Reply:
@v-admahe This change has been made